rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -187,6 +188,13 @@ impl FirmwareProcessor {
         manifest: &ImageManifest,
         img_bundle_sz: u32,
     ) -> CaliptraResult<ImageVerificationInfo> {
+        #[cfg(feature = "val-rom")]
```

Comment:
I don't think there is much of a point for the added complexity of the val verify environment if we can add the cfg feature switches in the lower level files. You could just cut out the actual verify call instead.

The reason I introduced that was because I was under the impression changes were to be limited to main.rs